### ISSUE TITLE
Toxtree : Some modules can not be run automatically

### ISSUE THREAD

**Issue by vedina** [ideaconsult/Toxtree.js#66]
Some modules can not be run altogether automatically  

![image](https://cloud.githubusercontent.com/assets/211190/4421981/6f8a5890-4588-11e4-8a57-e461a9b0f873.png)

but have to be run manually

![image](https://cloud.githubusercontent.com/assets/211190/4421985/751bf476-4588-11e4-8bef-39479bb12679.png)

Try [Toxtree with 111-96-6](http://apps.ideaconsult.net:8080/data/ui/toxtree)

**Comment by vedina** [comment]
Also gives error

````
Uncaught TypeError: Cannot read property 'indexOf' of undefined
````

**Comment by gonzomir** [comment]
This should be fixed together with #91.
